app/src/main/java/edu/uc/zhukv/droneradarmap/MapsActivity.kt

```diff
@@ -152,7 +157,7 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
             ) {
                 return
             }
-            mMap.isMyLocationEnabled = true;
```

Comment:
The semicolon was not needed here. -https://kotlinlang.org/docs/coding-conventions.html#formatting